docs/reverse-replication/ReverseReplicationUserGuide.md

```diff
@@ -142,6 +142,15 @@ The progress of files created per shard is also captured in the shard_file_proce
 
 To confirm that the records have indeed been written to the source database, best approach is to check the record count on the source database, if that matches the expected value. Note that verifying data takes more than just record count matching. The suggested tool for the same is [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator).
 
+#### Tracking which shards are lagging
+
+The following sample SQL gives the shards which are yet to catchup in the writer job. The SQL needs to be fired on the metatdata database. Replace the run_id with the relevant run identifier and the window interval with appropriate value.
+
+```
+select w.shard,r.shard,r.created_upto , w.file_start_interval from shard_file_create_progress r , shard_file_process_progress w where  r.run_id = w.run_id and r.shard = w.shard and
+TIMESTAMP_DIFF(r.created_upto , w.file_start_interval, SECOND)  > 10
```

Comment:
If we can order by higher lag first, that'd be a good to have.

Reply:
Done